softoff/meson.build

```diff
@@ -16,12 +16,16 @@ executable('pldm-softpoweroff',source,
            install: true,
            install_dir: get_option('bindir'))
 
-if get_option('systemd').allowed()
-  filesystem.copyfile(
-    'services/pldmSoftPowerOff.service',
-    'pldmSoftPowerOff.service',
+if get_option('systemd').enabled()
+  configure_file(input: 'services/pldmSoftPowerOff.service',
+                   output: 'pldmSoftPowerOff.service',
+                   copy: true,
+                   install_dir: systemd_system_unit_dir)
```

Comment:
we need just master change here, no need to put the downstream change.

Reply:
done